registers/data/nginx/default.conf

```diff
@@ -18,6 +18,8 @@ server {
 
   more_set_headers "Server: registers/0.1";
 
+  expires 0;
+  add_header Cache-Control "no-cache";
```

Comment:
What's the benefit of this?

Reply:
There isn't. By default, the PaaS CDN will cache things without our will, breaking the intended behaviour. We can manage that by telling NGINX to do the thing.

Reply:
Mmm, will that leak to the API users as well or the PaaS CDN resets headers?

Reply:
Not sure what you mean. The way clients can "cache" the data, is to store it upon receiving.

The headers will be required for the CDN not to cache.